docs/source/conf.py

```diff
@@ -79,18 +86,29 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-# html_static_path = ["../../images"]
-# html_extra_path = ["images"]
+html_static_path = []
 
 source_parsers = {".md": CommonMarkParser}
 source_suffix = [".rst", ".md"]
 
+nbsphinx_allow_errors = True
+html_show_sourcelink = False
+
+if os.path.exists(gitdir):
+    tag_refs = subprocess.check_output(["git", "tag", "-l", "v*"]).decode("utf-8").split()
+    tag_refs = natsorted(tag_refs)[-6:]
+    smv_tag_whitelist = r"^(" + r"|".join(tag_refs) + r")$"
+else:
+    smv_tag_whitelist = r"^v.*$"
+
+smv_branch_whitelist = r"^main$"
+
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3", None),
     "cudf": ("https://docs.rapids.ai/api/cudf/stable/", None),
     "distributed": ("https://distributed.dask.org/en/latest/", None),
     "torch": ("https://pytorch.org/docs/stable/", None),
-    "merlin-core": ("https://nvidia-merlin.github.io/core/", None),
+    "merlin-core": ("https://nvidia-merlin.github.io/main/core/", None),
```

Comment:
good catch - I'm guessing this was the issue with intersphinx mappings you were talking about yesterday